alacritty/src/display/mod.rs

```diff
@@ -1061,9 +1061,11 @@ impl Display {
     #[inline]
     fn damage_highlighted_hints<T: EventListener>(&self, terminal: &mut Term<T>) {
         let display_offset = terminal.grid().display_offset();
+        let last_line = terminal.screen_lines() - 1;
```

Comment:
```suggestion
        let last_line = terminal.topmost_line();
```

Reply:
Oh right…

Reply:
Maybe we want to name this viewport_top instead? Just to be explicit?

Reply:
Well, we can call it `last_screen_line`?

Reply:
Decided to go with just `last_visible_line`, I think that should be fine. I don't think there's anything *perfect*, so no point endlessly bikeshedding about variable naming.